ui/src/views/Clients/ViewClient/index.jsx

```diff
@@ -215,7 +215,7 @@ export default class ViewClient extends Component {
       }
 
       if (isNewClient) {
-        this.props.history.push({
+        this.props.history.replace({
```

Comment:
👍 This one looks good. The others should all have `.push`. In other words, the final diff should be:

```diff
diff --git a/ui/src/views/Clients/ViewClient/index.jsx b/ui/src/views/Clients/ViewClient/index.jsx
index 64ab92e47..5a5b853da 100644
--- a/ui/src/views/Clients/ViewClient/index.jsx
+++ b/ui/src/views/Clients/ViewClient/index.jsx
@@ -215,7 +215,7 @@ export default class ViewClient extends Component {
       }
 
       if (isNewClient) {
-        this.props.history.push({
+        this.props.history.replace({
           pathname: `/auth/clients/${encodeURIComponent(clientId)}`,
           state: { accessToken: result.data.createClient.accessToken },
         });
```

Reply:
> 👍 This one looks good. The others should all have `.push`. In other words, the final diff should be:
> 
> ```diff
> diff --git a/ui/src/views/Clients/ViewClient/index.jsx b/ui/src/views/Clients/ViewClient/index.jsx
> index 64ab92e47..5a5b853da 100644
> --- a/ui/src/views/Clients/ViewClient/index.jsx
> +++ b/ui/src/views/Clients/ViewClient/index.jsx
> @@ -215,7 +215,7 @@ export default class ViewClient extends Component {
>        }
>  
>        if (isNewClient) {
> -        this.props.history.push({
> +        this.props.history.replace({
>            pathname: `/auth/clients/${encodeURIComponent(clientId)}`,
>            state: { accessToken: result.data.createClient.accessToken },
>          });
> ```


do you mean to say, i only change one line of push() to replace() and leave the other lines with push() ?

Reply:
Correct.